R/barchart.R

```diff
@@ -450,11 +450,12 @@ Bar <- function(x,
             pt.segs <- NULL
             ind.show <- which(data.label.show[,i])
             data.label.text <- NULL
+            multi.color.labels <- multi.colors.within.series && length(unique(dlab.color)) > 1
```

Comment:
What happens when `length(unique(dlab.color)) == 1`? Do we still apply the colour somewhere?

Reply:
Yep, if there's only one font color, then the color is set in https://github.com/Displayr/flipStandardCharts/blob/master/R/barchart.R#L444. In that case we don't need this whole list so we try to avoid creating this if possible.